app/assets/javascripts/spam2.js

```diff
@@ -11,7 +11,12 @@ function table_main(id) {
 		"search": {
 			"regex": true
 		},
-		"scrollX": true
+		"scrollX": true,
+		"paging":false,
```

Comment:
Could you please format JS properly (with a tool)?
Thanks, @keshavsethi.

Reply:
Ya Sure, Thanks!